includes/User.php

```diff
@@ -130,12 +130,18 @@ class User implements JsonSerializable {
 	/** @name Cache variables */
 	//@{
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)